askama_shared/src/lib.rs

```diff
@@ -257,19 +258,71 @@ where
     vals.iter().map(|s| s.to_string()).collect()
 }
 
+/// Whitespace handling of the input source.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Strip {
+    /// Don't strip any spaces in the input.
+    None,
+    /// Remove a single single newline at the end of the input. This is the default.
+    Tail,
+    /// Remove all whitespaces at the front and back all lines, and remove empty lines.
```

Comment:
Missing `of`? Please remove periods at the end of the first line of docstrings.

Reply:
Fixed.